Develop

vanjamilicevic opened the pull request.

vanjamilicevic committed:
Created initial version of the app
- Added HTML
- Added responsive CSS and responsive CSS
- Added JS

vanjamilicevic committed:
Added README and screenshots for desktop/mobile

vanjamilicevic committed:
Enhanced mobile and added landscape

ghost commented:
![image](https://user-images.githubusercontent.com/90243506/135526717-9f798d88-4118-4bb7-8ba1-4d05b8c1331c.png)
See what happens with 'placeholders' when we have an error message.
The rest is good.

vanjamilicevic committed:
Fixed placeholders when we have an error message

 - Changed color on email input to #FF7979 when error hapens
 - Changed color on email input to return to #3D3B48
   when email is correct
 - Fixed missing part in password behavior when error hapens
